pkg/sql/exec_util.go

```diff
@@ -1057,6 +1057,20 @@ func golangFillQueryArguments(args ...interface{}) (tree.Datums, error) {
 				switch {
 				case val.IsNil():
 					d = tree.DNull
+				case val.Type().Elem().Kind() == reflect.String:
+					a := tree.NewDArray(types.String)
+					for i := 0; i < val.Len(); i++ {
+						if !val.Index(i).IsValid() {
```

Comment:
i think an invalid object doesn't need to be handled separately, so we can just remove this check. looks like the docs say that we can assume that a value returned by `.index(i)` is valid